test/cpp/test_tensor.cpp

```diff
@@ -421,78 +421,279 @@ TEST_F(TensorTest, TestBatchNorm1D) {
 }
 
 TEST_F(TensorTest, TestConv2D) {
-  int in_channels = 3;
-  int out_channels = 7;
+  int in_channels = 9;
+  int out_channels = 3;
   int kernel_size = 5;
   at::Tensor input =
-      at::rand({4, in_channels, 28, 28}, at::TensorOptions(at::kFloat));
-  at::Tensor weight =
-      at::rand({out_channels, in_channels, kernel_size, kernel_size},
-               at::TensorOptions(at::kFloat));
+      at::rand({4, in_channels, 32, 32}, at::TensorOptions(at::kFloat));
```

Comment:
Why 28 -> 32?

Reply:
I found a bug with input shape 32 in mkldnn backend while testing this which doesn't repro with other shapes. https://github.com/pytorch/pytorch/issues/25101 So I thought I'd use different input sizes so make sure the testing covers broader cases

Reply:
Haha, yes, I remember that. It showed up for us as well, and tweaking the size fixed it :)